src/static/js/modules/loan-comparison/common.js

```diff
@@ -21,8 +21,8 @@ common.scenarios = [
         label: 'down payment',
         intro: "Your down payment amount affects all aspects of your costs. Putting down less up front can be a good option for home buyers without a lot of cash on hand, but you’ll have higher monthly payments and pay more in interest and fees. This tool will help you get a sense of how much the difference in costs is likely to be, so you can make tradeoffs.",
         loanProps: [
-            {downpayment: 40000}, 
-            {downpayment: 20000}
+            {'downpayment-percent': 20},
+            {'downpayment-percent': 10}
         ],
         sharedProps: {},
```

Comment:
is `sharedProps` used anywhere? I couldn't find it, and it seems like we're defining `independentInputs` instead for the linked functionality